app/static/js/events/scheduler.js

```diff
@@ -919,6 +957,12 @@ function loadMicrolocationsToTimeline(day) {
     $("[data-toggle=tooltip]").tooltip("hide");
 
     if (isReadOnly()) {
+        _.each($microlocations, function ($microlocation) {
+            $microlocation = $($microlocation);
+            if ($microlocation.find('.microlocation-inner').html().trim() === "") {
```

Comment:
Instead of this, a more straightforward way would be to count the children.
`$microlocation.find('.microlocation-inner').children().length;`